test/core/derivatives/implementation/base.py

```diff
@@ -162,4 +162,13 @@ def sum_hessian(self) -> Tensor:
         Returns:
             the sum of hessians
         """
-        raise NotImplementedError
+        return
+
+    @abstractmethod
+    def hessian_is_zero(self) -> bool:
+        """Return whether the input-output Hessian is zero.
+
+        Returns:
+            `True`, if Hessian is zero, else `False`.
+        """
+        return
```

Comment:
`bool(None)` is `False`
So this is dangerous. Do you agree?

Reply:
Agree. But this `return statement` will never be evaluated due to the `@abstractmethod` marker.

Reply:
Resolved as above.